### PR TITLE
Add test to verify that ranges make progress while a minority is down.

### PR DESCRIPTION
**Pull request by bdarnell**
Stylistic updates to multiTestContext.

### CODE REVIEW

File: storage/client_test.go [hunk 1]
```diff
@@ -198,33 +209,64 @@ func (m *multiTestContext) addStore(t *testing.T) {
...
+		if err != nil {
+			m.t.Fatal(err)
+		}
+	}
+
+	// Wait for the replication to complete on all destination nodes.
+	util.SucceedsWithin(m.t, time.Second, func() error {
+		for _, dest := range dests {
+			// Use LookupRange(keys) instead of GetRange(raftID) to ensure that the
+			// snaphost has been transferred and the descriptor initialized.
```
Comment (tschottdorf) [inline storage/client_test.go]: s/phost/pshot/

**Comment by tschottdorf** [comment]
LGTM

**Comment by mrtracy** [comment]
LGTM

**Comment by spencerkimball** [comment]
LGTM

**Pull request merged**